unlock-app/src/components/interface/locks/Settings/forms/hooksComponents/GuildContractHook.tsx

```diff
@@ -0,0 +1,84 @@
+import { CustomComponentProps } from '../UpdateHooksForm'
+import { Select } from '@unlock-protocol/ui'
+import { useEffect, useState } from 'react'
+import { useUserGuilds } from '~/hooks/useUserGuilds'
+import Link from 'next/link'
+import {
+  useGetLockSettings,
+  useSaveLockSettings,
+} from '~/hooks/useLockSettings'
+
+export const GuildContractHook = ({
+  defaultValue,
+  lockAddress,
+  network,
+}: CustomComponentProps) => {
+  const [isSet, setIsSet] = useState(false)
```

Comment:
A bit confused what this is set state hook is used for?

Reply:
cleaned up